index.ts

```diff
@@ -0,0 +1,40 @@
+/* global posthog */
+/* eslint no-undef: "error" */
+
+import fetch from 'node-fetch'
+
+export async function setupPlugin(meta) {
+    runEveryMinute(meta)
+}
+
+export async function runEveryMinute({ config }) {
+    const timestamp = new Date().toISOString()
+    const capturePromises = [captureHeartbeat(timestamp)]
+    if (config.events.includes('heartbeat_api')) {
+        capturePromises.push(captureHeartbeatApi(timestamp, config.host, config.project_api_key))
+    }
+    await Promise.all(capturePromises)
+    console.log(`Captured: ${capturePromises.length}`)
```

Comment:
maybe console log debug timestamp too? Doesn't hurt to know when it was actually triggered in debug mode

Reply:
This was some initial debugging code but actually it wasn't very useful